conjure-java-core/src/integrationInput/java/com/palantir/product/SafeExternalLongExample.java

```diff
@@ -158,68 +165,68 @@ public Builder from(SafeExternalLongExample other) {
         }
 
         @JsonSetter("safeExternalLongValue")
-        public Builder safeExternalLongValue(long safeExternalLongValue) {
+        public Builder safeExternalLongValue(@Safe long safeExternalLongValue) {
             checkNotBuilt();
             this.safeExternalLongValue = safeExternalLongValue;
             this._safeExternalLongValueInitialized = true;
             return this;
         }
 
         @JsonSetter(value = "optionalSafeExternalLong", nulls = Nulls.SKIP)
-        public Builder optionalSafeExternalLong(@Nonnull Optional<? extends Long> optionalSafeExternalLong) {
+        public Builder optionalSafeExternalLong(@Nonnull Optional<? extends @Safe Long> optionalSafeExternalLong) {
```

Comment:
this is......technically correct i think

Reply:
👍 Looks good to me